manifests/params.pp

```diff
@@ -23,6 +23,53 @@
   $ntp            = true
   $force          = false
 
+  # Determine if client needs manual sudo configuration or not
+  # RHEL <=6.5 requires manual configuration
+  # RHEL 6.6 includes automatic sudo configuration
+  # RHEL 7.0 requires manual confifuration
+  # RHEL >=7.1 includes automatic sudo configuration
+  case $::osfamily {
+    RedHat: {
+      case $::operatingsystem {
+        'Fedora': {
+          if (versioncmp($::operatingsystemrelease, '21') >= 0) {
+            $needs_sudo_config = '0'
```

Comment:
I'd prefer if these were actual booleans, to be consistent with what's elsewhere